.vscode/settings.json

```diff
@@ -1,5 +1,9 @@
 {
     "go.testTimeout": "10s",
+    "go.buildFlags": [
+        // Hardcoded the runme version because VSCode couldn't resolve it.
+        "-ldflags=-X 'github.com/stateful/runme/internal/version.BuildVersion=2.0.0'",
```

Comment:
@pastuxso wondering if we should set it something more fixture-like such as `1.2.3`. I think it'll stand out more since we're now officially on 2.x.x.